pkg/giturl/giturl_test.go

```diff
@@ -187,3 +189,76 @@ func TestGenerateLink(t *testing.T) {
 		})
 	}
 }
+
+func TestUpdateLinkLineNumber(t *testing.T) {
+	t.Parallel()
+
+	type args struct {
+		link    string
+		newLine int64
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
```

Comment:
__optional__: Could be nice to have a bitbucket "link change" (aka no change to the link) test case.